core/platform/datastore/gae_datastore_services.py

```diff
@@ -312,6 +311,10 @@ def __instancecheck__(cls, other):
 
     # We use ‘ignore[misc]’ because MockDatetime subclasses a class created
     # dynamically. These dynamic classes are not supported by mypy.
+    # This can be fixed after py3 migration as this error is resolved when
```

Comment:
Please add a TODO and issue since this refers to a future task.

Reply:
Done